org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CodeActionHandlerTest.java

```diff
@@ -83,11 +83,10 @@ public void testCodeAction_removeUnusedImport() throws Exception{
 		params.setContext(new CodeActionContext(Arrays.asList(getDiagnostic(Integer.toString(IProblem.UnusedImport), range))));
 		List<Either<Command, CodeAction>> codeActions = getCodeActions(params);
 		Assert.assertNotNull(codeActions);
-		Assert.assertEquals(4, codeActions.size());
+		Assert.assertTrue(codeActions.size() >= 3);
 		Assert.assertEquals(codeActions.get(0).getRight().getKind(), CodeActionKind.QuickFix);
 		Assert.assertEquals(codeActions.get(1).getRight().getKind(), CodeActionKind.QuickFix);
 		Assert.assertEquals(codeActions.get(2).getRight().getKind(), CodeActionKind.SourceOrganizeImports);
-		Assert.assertEquals(codeActions.get(3).getRight().getKind(), JavaCodeActionKind.SOURCE_OVERRIDE_METHODS);
```

Comment:
why is this code action no longer returned?

Reply:
it still returned. I refactor this test case because this test method name is `removeUnusedImport`. Checking all code action list is just a byproduct. Every time, a new source action is added, we need fix this case. so i refactor this test case and let it focus on checking `removeUnusedImport` related code action only.